tests/js/blocks/content-helper/structure.test.js

```diff
@@ -72,25 +72,6 @@ describe( 'Content Helper', () => {
 		);
 	} );
 
-	test( 'should show error message when WordPress REST API returns the error', async () => {
```

Comment:
Removed this test because as of now this is extra i.e. we are treating parsely api errors and wp api errors in same way.